yroots/subdivision.py

```diff
@@ -27,9 +27,9 @@
 def solve(funcs, a, b, rel_approx_tol=1.e-15, abs_approx_tol=1.e-12,
           max_cond_num=1e5, good_zeros_factor=100, min_good_zeros_tol=1e-5,
           check_eval_error=True, check_eval_freq=1, plot=False,
-          plot_intervals=False, deg=20, target_deg=3, max_level=999,
-          return_potentials=False, method='svd'):
-    '''
+          plot_intervals=False, deg=None, target_deg=None, max_level=999,
```

Comment:
Why do we leave deg and target_deg in here? I'm guessing just to give the user some more fine-tuned control, but we give defaults inside the function

Reply:
I suppose that we could remove them, that's true. I just kept them in case the user wanted to have more fine-tuned control, and I had forgotten that previously we didn't even have them as function arguments.